_posts/v2/2015-10-13-country-fact-sheets.md

```diff
@@ -0,0 +1,56 @@
+---
+permalink: "/country-fact-sheets.html"
+layout: body
+title: Country Fact Sheets API
+published: true
+---
+
+#Country Fact Sheets API
+
+{% include country-fact-sheets-tabs %}
+
+##Resource URL
+
+{% include country-fact-sheets-query.html %}
+
+##Search Parameters for country fact sheets
+
+###keyword
+
+Returns country fact sheets for a match in the **title** or **content_html** fields.
+
+    {{ site.webservices_baseurl }}/country_fact_sheets/search?api_key={your key}&q={keyword}
+
+**_Example_**
+
+{% include country-fact-sheets-query-keyword.html %}
+
+###countries
+
+Returns country fact sheets for a specific country based on [ISO alpha-2 country codes](http://www.iso.org/iso/home/standards/country_codes/country_names_and_code_elements.htm). Enter multiple terms by seperating with a comma.
+
+    {{ site.webservices_baseurl }}/country_fact_sheets/search?api_key={your key}&countries={country code}
+
+**_Example_**
+
+{% include country-fact-sheets-query-keyword-country.html %}
+
+###size + offset
+
+The **size** parameter allows you to configure the number of results to be returned up to a maximum of 100. The **offset** parameter defines the offset from the first result you want to fetch. Unless specified the API returns 10 results at a time.
+
+**_Example_**
+
+{% include country-fact-sheets-query-size.html %}
+
+###Return Values
+
+| Field           | Description                                                     |
+| --------------- | --------------------------------------------------------------- |
+| id              | Unique identifier assigned to the fact sheet.                   |
+| title           | Unique identifier assigned to the fact sheet.                   |
+| content_html    | Unique identifier assigned to the fact sheet.                   |
+| published_date  | Country category(ies) assigned to the report.                   |
+| country         | Country category(ies) assigned to the report.                   |
+| source          | Abstract of the report’s content.                               |
+| topic           | Date when the report is no longer valid.                        |
```

Comment:
Changes look good except for this table. I guess you should be able to describe what each field contains.